services/pfs.py

```diff
@@ -100,11 +103,13 @@ def get_output(data):
 
         plugin.update(
             name='Java Runtime Environment',
-            manualInstallationURL='https://support.mozilla.org/kb/use-java-plugin-to-view-interactive-content')
+            manualInstallationURL='https://support.mozilla.org/kb/use-java-'
+                                  'plugin-to-view-interactive-content')
```

Comment:
can we put that into a constant?

Reply:
https://github.com/mozilla/addons-server/pull/6583/commits/53e16a83bcda6a6f89369619a3fe1523e392f63e